lego/apps/content/models.py

```diff
@@ -78,3 +84,7 @@ def __str__(self):
     @property
     def comment_target(self):
         return f"{self._meta.app_label}.{self._meta.model_name}-{self.pk}"
+
+    @property
+    def reaction_target(self):
```

Comment:
Should we combine comment_target and reaction_target? I can't imagine a case where they would be different.

Reply:
Not a bad idea, I'll rename it to `content_target`

Reply:
Hm, we also need to keep in mind that renaming it is not backwards compatible, so the backend needs to be deployed together with the frontend. If one is deployed before the other, comments will break.